_drafts/2017-xx-xx-patentsview.Rmd

```diff
@@ -0,0 +1,244 @@
+---
+title: "Accessing patent data with the patentsview package"
+name: patentsview
+author:
+  - name: Chris Baker
+    url: http://github.com/crew102
+date: '2017-08-20'
```

Comment:
Tues Sept 12th is the first date with an open slot for weekly main blog posts (temporarily suffering from an embarrassment of riches). Once you make changes and give the go-ahead, we can post this earlier if another post drops out of the pipeline.